src/components/ButtonCircleToggle/ButtonCircleToggle.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useRef } from 'react';
+import classnames from 'classnames';
+import ButtonCircle from '../ButtonCircle';
+import { useToggleState } from 'react-stately';
+import { useToggleButton } from 'react-aria';
+
+import { DEFAULTS, STYLE } from './ButtonCircleToggle.constants';
+import { Props } from './ButtonCircleToggle.types';
+import './ButtonCircleToggle.style.scss';
+
+const ButtonCircleToggle: FC<Props> = (props: Props) => {
+  const {
+    children,
+    className,
+    ghost = DEFAULTS.GHOST,
+    outline = DEFAULTS.OUTLINE,
+    disabled = DEFAULTS.DISABLED,
+    ...otherProps
+  } = props;
+
+  const ref = useRef();
+  const state = useToggleState(props);
+  const { buttonProps } = useToggleButton(props, state, ref);
+
+  if (ghost === false && outline === false) {
```

Comment:
Is this a toggle button specific problem or should we have this warning for the `ButtonCircle` as well?

I think we should not show this error in production build so please add the `process.env.NODE_ENV !== "production"` condition as well
```suggestion
if (process.env.NODE_ENV !== "production")
   if (ghost === false && outline === false) {
```

if the build fails because of this extra condition then you can remove it for now

Reply:
we should not have ths warning for buttonCircle; only for this toggle

Reply:
I think it's fine to add this warning in production. It's there for a reason to prevent the consumers to use this component in the wrong way.